includes/misc-functions.php

```diff
@@ -1051,6 +1051,23 @@ function edd_is_promo_active() {
 	return false;
 }
 
+/**
+ * Gets the date that this EDD install was activated (for new installs).
+ * For existing installs, this option is added whenever the function is first used.
+ *
+ * @since 2.11.4
+ * @return string The timestamp when EDD was activated.
+ */
+function edd_get_activation_date() {
+	$activation_date = get_option( 'edd_activation_date', '' );
+	if ( ! $activation_date ) {
+		update_option( 'edd_activation_date', time() );
+		$activation_date = get_option( 'edd_activation_date', '' );
```

Comment:
```suggestion
		$activation_date = time();
		update_option( 'edd_activation_date', $activation_date );
```

What do you think about this instead of calling `get_option()` ?

Reply:
Yes, thank you, I like that better for sure.